Source/WebGPU/WebGPU/RenderBundleEncoder.mm

```diff
@@ -186,13 +222,28 @@
 
 Ref<RenderBundle> RenderBundleEncoder::finish(const WGPURenderBundleDescriptor& descriptor)
 {
+    if (!m_currentCommandIndex)
+        return RenderBundle::createInvalid(m_device);
+
     auto commandCount = m_currentCommandIndex;
     m_currentCommandIndex = 0;
 
     if (!m_indirectCommandBuffer) {
-        m_icbDescriptor.maxVertexBufferBindCount = m_device->maxBuffersPlusVertexBuffersForVertexStage();
+        m_icbDescriptor.maxVertexBufferBindCount = m_device->maxBuffersPlusVertexBuffersForVertexStage() + 1;
```

Comment:
Could you a comment here why `+ 1`?

Reply:
oh because the dynamic offsets get set to `index = m_device->maxBuffersPlusVertexBuffersForVertexStage()` and if I set them to `m_device->maxBuffersPlusVertexBuffersForVertexStage() - 1` then I need to change the WGSL compiler.

But that is kind of ugly, maybe I can clean that up